packages/admin-ui/src/components/Sidebar/components/SidebarItemSkeleton.tsx

```diff
@@ -13,44 +24,33 @@ export function SidebarItemSkeleton() {
     <Box
       csx={{
         width: SCALES.COLLAPSIBLE_AREA_WIDTH,
-        padding: '1.5rem 0.5rem',
+        padding: '$s',
       }}
     >
-      {Array.from(new Array(2)).map((_, index) => (
-        <Set
-          key={`sidebar-section-${index}`}
-          spacing={1}
-          orientation="vertical"
+      <Box csx={{ size: '100%', margin: '$s', paddingY: '$l' }}>
+        <Skeleton
+          key="section-title"
           csx={{
-            width: SCALES.SIDEBAR_SECTION_WIDTH,
-            paddingBottom: 8,
+            width: '5rem',
+            height: '1.5rem',
           }}
-        >
-          <Box
-            csx={{
-              minHeight: '1rem',
-              width: '6rem',
-              paddingBottom: '0.8125rem',
-            }}
-          >
+        />
+      </Box>
+      {Array.from(new Array(7)).map((_, index) => {
+        const width = skeletonWidth[index as SkeletonIndex]
+
+        return (
+          <Box csx={{ size: '100%', padding: '$s' }}>
             <Skeleton
-              key={`section-title-${index}`}
+              key={`section-item-${2 + index}`}
```

Comment:
Why `2`?

Reply:
It was necessary on the previous version of the skeleton. I'll change, good catch!